app/views/layouts/_header.html.erb

```diff
@@ -51,7 +51,7 @@
             <a class="dropdown-item" href="<%= user_groups_path(current_user) %>"><%= t("layout.header.my_groups") %></a>
             <a class="dropdown-item" href="<%= notifications_path(current_user) %>"><%= t("layout.header.notifications") %></a>
             <div class="dropdown-divider"></div>
-            <a class="dropdown-item" rel="nofollow" data-method="delete" href="<%= destroy_user_session_path %>"><%= t("sign_out") %></a>
+            <a class="dropdown-item" rel="nofollow" data-method="delete" href="<%= link_to "Logout", destroy_user_session_path, :method => :delete %>"><%= t("sign_out") %></a>
```

Comment:
Here you are trying to embed an acnhor tag inside anchor tag, try following this guide to get basic understanding about the rails syntax https://guides.rubyonrails.org/getting_started.html